app/models/ems_refresh.rb

```diff
@@ -55,6 +55,10 @@ def self.queue_refresh(target, id = nil, sync = false)
     end
   end
 
+  def self.queue_event_for_refresh(event, ems)
```

Comment:
You'll want to keep the method names consistent, typically the naming pattern for a queued method named `refresh_new_target` would be `queue_refresh_new_target` that way its more obvious what is going to get queued.